tools/c7n_azure/c7n_azure/output.py

```diff
@@ -89,8 +89,13 @@ def upload(self):
                         blob_name,
                         os.path.join(root, f))
                 except AzureHttpError as e:
-                    self.log.error("Error writing output. Confirm output storage URL is correct "
-                                   "and that 'Storage Blob Contributor' role is assigned. \n" +
+                    if e.status_code == 403:
+                        self.log.error("Access error writing output. "
```

Comment:
I prefer if we do something like `Access Error: Storage Blob Data Contributor Role is required to write to Blob Storage.`